website/docs/docs/building-a-dbt-project/metrics.md

```diff
@@ -90,6 +92,85 @@ metrics:
 
 </File>
 
+<VersionBlock firstVersion="1.2">
+
+### Available properties
+
+| Field       | Description                                                 | Example                         | Required? |
+|-------------|-------------------------------------------------------------|---------------------------------|-----------|
+| name        | A unique identifier for the metric                          | new_customers                   | yes       |
+| model       | The dbt model that powers this metric                       | dim_customers                   | <VersionBlock firstVersion="1.2">yes (no for `expression` metrics)</VersionBlock><VersionBlock lastVersion="1.1">yes</VersionBlock> |
```

Comment:
Idea in making this change is that we can have just one table of "Available properties," rather than duplicating it here + below (beneath `<VersionBlock lastVersion="1.1">`)

Reply:
Accepted changes!

Reply:
Not quite. Here's a commit with what I mean: https://github.com/dbt-labs/docs.getdbt.com/commit/54b9b818db0121d2e8fea92690d169933585cdaf

Reply:
@callum-mcdata Still need to delete the duplicated version of this table below as well:

<img width="556" alt="Screenshot 2022-07-14 at 01 50 40" src="https://user-images.githubusercontent.com/13897643/178856008-38b1fdec-0472-4a83-abdf-2967fb35f377.png">

Reply:
Odd - when I spun it up on my local there was only one table 🤔  I wonder if I messed up on the merge conflict from current. Let me fix now!